numpy/lib/index_tricks.py

```diff
@@ -368,8 +368,8 @@ def __getitem__(self, key):
                         if len(vec) == 3:
                             trans1d = int(vec[2])
                         continue
-                    except Exception:
-                        raise ValueError("unknown special directive")
+                    except Exception as e:
+                        raise ValueError("unknown special directive {!r}".format(item)) from e
```

Comment:
```suggestion
                        raise ValueError(
                            "unknown special directive {!r}".format(item)
                        ) from e
```
Style nit - 80 char limit